generator.yaml

```diff
@@ -925,5 +983,6 @@ ignore:
       - InstanceGroups
```

Comment:
Can you remove InstanceGroups and InstanceGroupNames and regenerate

Reply:
reason?

Reply:
done